### DISCUSSION TITLE
FR - dont hide entry type name in card mode if some field values are displayed

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#15055] [Ideas] [answered]
With many entry type available to select in the matrix fields, it is easy to loose track of what type of content specific card represents, even if we give it unique color and icon.
There should be an option to display entry type name at all times - right now it wont show up when some field values are displayed in the card.

**Comment by ryanleichty** [comment]
We're running into this issue as well 🙏

**Comment by thupsi** [comment, marked as answer]
Maybe https://github.com/craftcms/cms/pull/15283 will solve this?

**Reply by brandonkelly** [reply]
Yeah, @i-just let’s make sure “Entry Type” is one of the selectable attributes for entries.

**Reply by ryanleichty** [reply]
That would solve it for us 👍
